engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -4,248 +4,222 @@ keywords: 'container, storage, driver, AUFS '
 title: Docker and AUFS in practice
 ---
 
-AUFS was the first storage driver in use with Docker. As a result, it has a
-long and close history with Docker, is very stable, has a lot of real-world
-deployments, and has strong community support. AUFS has several features that
-make it a good choice for Docker. These features enable:
+AUFS a *union filesystem*. The `aufs` storage driver is the default storage
+driver used for managing images and layers on Docker, for Ubuntu and Debian.
+AUFS is the most mature storage driver used by Docker. It provides fast
+container startup time, as well as efficient usage of memory and storage.
 
-- Fast container startup times.
-- Efficient use of storage.
-- Efficient use of memory.
+If your Linux kernel is version 4.0 or higher, and you use Docker CE, consider
+using the newer [overlay2](overlayfs-driver.md){: target="_blank" class="_" },
+which has potential performance advantages over the `aufs` storage driver.
 
-Despite its capabilities and long history with Docker, some Linux distributions
- do not support AUFS. This is usually because AUFS is not included in the
-mainline (upstream) Linux kernel.
+> **Note**: Even though AUFS is the default storage driver, it is not supported
+> on some distributions and Docker editions. See [Prerequisites](#prerequisites)
+> for more information.
 
-The following sections examine some AUFS features and how they relate to
-Docker.
+## Prerequisites
 
-## Image layering and sharing with AUFS
+- For Docker CE, AUFS is supported on Ubuntu and Debian.
```

Comment:
For debian support, see my earlier comment (they dropped support for aufs, and no longer offer the required packages for their newer kernels)